package.json

```diff
@@ -7,7 +7,7 @@
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
     "build": "webpack",
-    "dev": "webpack serve --open",
```

Comment:
the `--open` param no need here, because the `devServer.open: true` is already defined  in Webpack config